pkgs/by-name/ol/ollama/package.nix

```diff
@@ -101,12 +101,12 @@ let
   };
 
   cudaToolkit = buildEnv {
-    name = "cuda-toolkit";
-    ignoreCollisions = true; # FIXME: find a cleaner way to do this without ignoring collisions
+    name = "cuda-merged";
     paths = [
-      cudaPackages.cudatoolkit
-      cudaPackages.cuda_cudart
-      cudaPackages.cuda_cudart.static
+      (lib.getBin (cudaPackages.cuda_nvcc.__spliced.buildHost or cudaPackages.cuda_nvcc))
+      (lib.getLib cudaPackages.cuda_cudart)
+      (lib.getOutput "static" cudaPackages.cuda_cudart)
```

Comment:
what's the advantage of `(lib.getOutput "static" cudaPackages.cuda_cudart)` over `cudaPackages.cuda_cudart.static`?

Reply:
The advantage of `getX` in general is that if we refactor the generator (`cuda-find-redist-features`) and, say, merge `bin` into `out`, the `getBin` consumers don't have to adapt.

`getOutput "static"` specifically is maybe somewhat useless: it allows us either to merge `static` into `out` (why would we; makes more sense to fallback to lib even), or accommodate removing static libraries altogether...  but then there's a matter of limiting the public interface